### ISSUE TITLE
Add config for Facebook signin via Cognito Identity

### ISSUE THREAD

**Issue by adrianhall** [awslabs/aws-amplify-serverless-plugin#14]
The config is:

```
"FacebookSignin": {
    "Permissions": "public_profile",
    "AppId": "<your-facebook-appid>"
}
```

**Comment by adrianhall** [comment]
The app ID is stored in the SupportedLoginProviders within the identity pool meta data under the `graph.facebook.com` key.

**Comment by adrianhall** [comment]
Implemented with PR #32
